h/static/scripts/directive/about-this-version-dialog.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+/**
+ * @ngdoc directive
+ * @name aboutThisVersionDialog
+ * @description Displays product version and environment info
+ */
+// @ngInject
+module.exports = function () {
+  return {
+    bindToController: true,
+    controllerAs: 'vm',
+    // @ngInject
+    controller: function ($scope, $window, settings, crossframe) {
+      this.userAgent = $window.navigator.userAgent;
+      this.version = settings.release;
+
+      $scope.$watchCollection(
+        function () {
+          return crossframe.frames;
+        },
+        function (frames) {
+          if (!frames || frames.length <= 0) {
```

Comment:
`frames.length` can never be `< 0` and frames is guaranteed to be an array, so this can be simplified to `if (frames.length === 0)`